site/access-control.xml

```diff
@@ -275,11 +275,16 @@ loopback_users = none
           installation). With this authorisation backend, topic
           authorisation as optional: you don't need to whitelist any
           exchanges. To use topic authorisation therefore you need to opt in
-          and define topic permissions for one or more exchanges.  For details please see the <a
+          and define topic permissions for one or more exchanges. For details please see the <a
           href="man/rabbitmqctl.1.man.html#Access%20control">Access
           Control section</a> of the <a
           href="man/rabbitmqctl.1.man.html">rabbitmqctl(1) man
-          page</a>.
+          page</a>. Note the default authorisation backend supports variable expansion
+          in the permission regular expression.
+          Three variables are supported: <code>username</code>, <code>vhost</code>,
+          and <code>client_id</code> (only when using MQTT). For example, the permission
+          <code>^{username}-.*</code> would be expanded to <code>^tonyg-.*</code>
+          if <code>tonyg</code> was the connected user.
```

Comment:
This may make it clearer that those variables are MQTT-only:

----

When using the MQTT protocol, three variables are supported: `<code>username</code>`, `<code>vhost</code>`, and `<code>client_id</code>`. For example, when `<code>tonyg</code>` is the connected user, the permission `<code>^{username}-.*</code>` is expanded to `<code>^tonyg-.*</code>`.